sql/select.go

```diff
@@ -468,13 +256,13 @@ func (v *indexInfo) init(s *scanNode) {
 
 // analyzeRanges examines the range map to determine the cost of using the
 // index.
-func (v *indexInfo) analyzeRanges(m qvalueRangeMap) {
+func (v *indexInfo) analyzeRanges(exprs []parser.Exprs) {
 	if !v.covering {
 		return
 	}
 
-	v.makeStartInfo(m)
-	v.makeEndInfo(m)
+	v.makeStartInfo(exprs)
+	v.makeEndInfo(exprs)
```

Comment:
the bodies of these methods are also pretty dense and very similar. a small comment on each one would go a long way in clarifying their differences